waspc/cli/Command/Compile.hs

```diff
@@ -0,0 +1,35 @@
+module Command.Compile
+    ( compile
+    ) where
+
+import           Control.Monad.IO.Class (liftIO)
+import           Data.List              (find, isSuffixOf)
+import qualified Path                   as P
+
+import qualified Common
+import           CompileOptions         (CompileOptions (..))
+import qualified Lib
+import           StrongPath             (Abs, Dir, Path, (</>))
+import qualified StrongPath             as SP
+import qualified Util.IO
+
+
+-- | Compiles Wasp source code in waspRoot directory and generates a project in given outDir directory.
+compile :: Path Abs (Dir Common.WaspProjectDir) -> Path Abs (Dir Lib.ProjectRootDir) -> IO (Either String ())
+compile waspRoot outDir = do
```

Comment:
what is waspRoot? Maybe add a comment for that, wasn't clear for me.

Reply:
Good point, renamed to waspProjectDir!